src/server/services/wfs/qgswfsgetfeature.cpp

```diff
@@ -75,7 +75,7 @@ namespace QgsWfs
                           QgsWfsParameters::Format format, int prec, QgsCoordinateReferenceSystem &crs,
                           QgsRectangle *rect, const QStringList &typeNames );
 
-    void setGetFeature( QgsServerResponse &response, QgsWfsParameters::Format format, QgsFeature *feat, int featIdx,
+    void setGetFeature( QgsServerResponse &response, QgsWfsParameters::Format format, const QgsFeature &feature, int featIdx,
```

Comment:
Just a totally unrelated note to express my joy everytime I read the `setGet...` :)

Reply:
`setGetFeatureResponseContent` could be more appropriate